### DISCUSSION TITLE
Can't install the configuration yaml

### DISCUSSION THREAD

**Discussion by wBrhy2** [emporia-vue-local/esphome#232] [Q&A] [answered]
Hello, I'm sure this is me being a doofus, it may be more of an esphome issue than a emporia-vue-local issue, but here goes. I've done the initial install on my device (as I've done with other esp32s before), and then I entered the yaml code in the edit window, reading it and changing what needed to be changed as per comments. However, I get an error when I try to install it and it fails before reading any of it. 

```
INFO ESPHome 2023.10.6
INFO Reading configuration /config/esphome/esphome-web-d8f3c0.yaml...
ERROR Error while reading config: Invalid YAML syntax:

while parsing a block mapping
  in "/config/esphome/esphome-web-d8f3c0.yaml", line 1, column 1:
    esphome:
    ^
expected <block end>, but found '<block mapping start>'
  in "/config/esphome/esphome-web-d8f3c0.yaml", line 24, column 2:
     services:
     ^
```

Is this saying that my first line, "esphome:" (which is pretty standard afaik) is causing an issue?

Thanks!

*edit 1* added ESPHome version.
*edit 2* I made sure there was no tab or hidden character by erasing the spaces in front of line 24 and adding the spaces manually, no luck.

**Comment by flaviut** [comment, marked as answer]
Can you please post your entire config? I can tell you with certainty that the error indicates you have the wrong number of spaces in front of line 24, but can't help besides that.

**Reply by wBrhy2** [reply]
Sure thing, here it is. I think you may be right, I thought because of the grey line in the yaml editor being in the right place that it meant it was properly indented, but it does seem like I had it 1 extra space over to than "encryption".

edit: removed my code because it's pointless, that's exactly it. The two errors were the 1 space missing on the services line.

It seems that when copy pasting the YAML I somehow created indentation errors? Fixed some other ones. Probably from copying in blocks and not all at once actually.

Thanks!
